utilities/test_suite/HOST/Tensor_host_audio.cpp

```diff
@@ -0,0 +1,206 @@
+/*
+Copyright (c) 2019 - 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "../rpp_test_suite_audio.h"
+
+int main(int argc, char **argv)
+{
+    // Handle inputs
+    const int MIN_ARG_COUNT = 6;
+    if (argc < MIN_ARG_COUNT)
+    {
+        printf("\nImproper Usage! Needs all arguments!\n");
+        printf("\nUsage: ./Tensor_host_audio <src folder> <u8 = 0 / f16 = 1 / f32 = 2 / u8->f16 = 3 / u8->f32 = 4 / i8 = 5 / u8->i8 = 6> <case number = 0:0> <test type 0/1> <numRuns> <batchSize> <dst folder>\n");
+        return -1;
+    }
+
+    char *src = argv[1];
+    int inputBitDepth = atoi(argv[2]);
+    int testCase = atoi(argv[3]);
+    int testType = atoi(argv[4]);
+    int numRuns = atoi(argv[5]);
+    int batchSize = atoi(argv[6]);
+    char *dst = argv[7];
+
+    if (testType == 0 && batchSize != 3)
+    {
+        cout << "Error! QA Mode only runs with batchsize 3" << endl;
```

Comment:
why is this a requirement

Reply:
@rrawther Since QA mode is designed only for Golden Output checks for sizes 50x50, 100x100 and 150x150 images. For Audio, similarly we have 3 samples to do our pixelwise QA check.